core/src/main/java/org/infinispan/container/EntryFactoryImpl.java

```diff
@@ -226,21 +217,22 @@ private CacheEntry getFromContext(InvocationContext ctx, Object key) {
       return cacheEntry;
    }
 
-   private InternalCacheEntry getFromContainer(Object key, boolean ignoreOwnership, boolean writeOperation) {
-      final boolean isLocal = distributionManager == null || distributionManager.getLocality(key).isLocal();
-      if (isLocal || ignoreOwnership) {
+   private CacheEntry getFromContainer(Object key, boolean shouldLoad, boolean writeOperation) {
```

Comment:
Then maybe you need to change the callers as well? All of them use a variable called `isOwner`.